### DISCUSSION TITLE
MODIS 8-days composition geolocation

### DISCUSSION THREAD

**Discussion by thaimunhoz** [microsoft/PlanetaryComputer#299] [Q&A] [answered]
Hi, I'm trying to create a mosaic using MODIS 8-days composition images. However, the final mosaic is presenting a geolocation error related to the orthorectification. Do you guys know if there is a problem with the collection or maybe some tip related to the CRS.

Thanks in advance!

![image](https://github.com/microsoft/PlanetaryComputer/assets/80410170/f73f15fa-1d4f-4bbf-a0d3-befc40a59dc1)
![image](https://github.com/microsoft/PlanetaryComputer/assets/80410170/e7fce965-55b1-4ffb-a311-74699248febb)

This is my script:

`
search = catalog.search(
        collections=["modis-09A1-061"],
        bbox=bbox,
        datetime= start_date + "/" + end_date,)
    
items = search.get_all_items()

stack = stackstac.stack(items, epsg = 4674, bounds_latlon=bbox, resampling=rasterio.enums.Resampling.bilinear,
                       rescale=True,)

nir, swir = stack.sel(band = "sur_refl_b02"), stack.sel(band="sur_refl_b05")
ndwi = ((nir - swir) / (nir + swir)).clip(0.0, 1) # NDWI varia de [0-1], para evitar a presença de ruído, excluí-se os pixels com valores abaixo de 0.08
ndwi = ndwi.where(ndwi > 0.0, other=np.nan)
by_year = ndwi.groupby("time.month").apply(stackstac.mosaic, dim="time", nodata=np.nan)
by_year.rio.to_raster("_process" + '/' + '3NDWI_' + str(month) + '.tif',  tiled=True)`

**Comment by TomAugspurger** [comment, marked as answer]
I haven't had a chance to check, but I know that `stackstac.stack` has an `xy_coords="center"/"topleft"` parameter. Would setting that to `center` will do the right thing? I'm not sure whether things are off more more than a pixel or not.
